app/src/main/res/values/colors.xml

```diff
@@ -53,6 +53,7 @@
     <color name="search_suggestion_indicator_icon_bookmark_color_light_theme">#0060df</color>
     <color name="preference_section_header_normal_theme">@color/accent_bright_light_theme</color>
     <color name="menu_item_button_light_theme">@color/accent_bright_light_theme</color>
+    <color name="site_permissions_caption_color_light_theme">@color/photonDarkGrey05</color>
```

Comment:
Hi, I will ask UX about these colors, but I suspect we already have appropriate design system colors for this and don't need to add new color variables.

Reply:
Hey @gabrielluong,

Thanks for taking a look! I had the same thought earlier when looking at the rebase. I left a comment over on the issue (https://github.com/mozilla-mobile/fenix/issues/16395#issuecomment-1025928342), but I'm thinking maybe we want fx_mobile_text_color_secondary?

In any case, I'll go ahead and fix this up and push again once we have the feedbacks from UX.

Reply:
Yes, that's what I am expecting, but I will have @cwzilla confirm.

Reply:
Confirmed with @cwzilla that Secondary Text is what you want here.

Reply:
Great, thanks Gabriel! I'll get that switched over tonight.